packages/block-editor/src/components/block-canvas/index.js

```diff
@@ -49,7 +45,12 @@ export function ExperimentalBlockCanvas( {
 						width: '100%',
 					} }
 				>
-					{ children }
+					<EditorStyles
+						styles={ styles }
+						scope=".editor-styles-wrapper"
+					>
+						{ children }
+					</EditorStyles>
```

Comment:
I guess this is sort of a breaking change, although <EditorStyles> is experimental.

Maybe there are other ways to achieve the same without the React context.

Reply:
In what way is it a breaking change?

Reply:
Because the component now accepts children, and blocks must be rendered as children. But this component is experimental so maybe it's ok. Worst case some styles are not applied.